kolibri/core/assets/src/views/CoreBanner.vue

```diff
@@ -57,9 +58,22 @@
         bannerClosed: false,
       };
     },
+    created() {
+      document.addEventListener('focusin', this.focusChange);
+    },
+    beforeDestroy() {
+      document.removeEventListener('focusin', this.focusChange);
+    },
     methods: {
       toggleBanner() {
         this.bannerClosed = !this.bannerClosed;
+        this.previouslyFocusedElement.focus();
+      },
+      focusChange(e) {
+        // We need the element prior to the close button
+        if (this.$refs.close_button && e.target != this.$refs.close_button.$el) {
```

Comment:
This extra check for `$refs.close_button` can make this feature a little confusing. For example, if I do this:

1. At Sign-In screen, click "Close" on demo bar right away.
2. Use my keyboard to focus on the "Create Account" button 
3. Press "More info" on demo bar
4. Then focus returns to the username bar.

Do you think you can update this to always have this behavior:

```
Given The demo banner is in closed state
When I focus on an element `$el` on the sign-in page
	And I re-open the demo banner
	And I re-close the demo banner
	Then focus is returned to `$el`.
```

Reply:
okay I understood I will do it.